src/workingtitle-vcockpits-instruments-navsystems-g3000/html_ui/WorkingTitle/Pages/VCockpit/Instruments/NavSystems/AS3000/AS3000_Common.js

```diff
@@ -0,0 +1,300 @@
+class AS3000_MapElement extends MapInstrumentElement {
```

Comment:
I'd be interested in how much of this we can pull out into a common space and put into Navsystems.js or CommonPFD_MFD.js so that it can be used by any nav system.   I'm sure features will vary between units, but I suspect the logic will be highly similar.

That's not a change requested for this PR, just something I think we should think about down the road.